app/src/main/java/org/greenstand/android/TreeTracker/view/SelectableItemDetail.kt

```diff
@@ -82,12 +86,29 @@ fun SelectableImageDetail(
                         .clip(RoundedCornerShape(10.dp)),
                 )
             }
-            painter?.let {
-                Image(
-                    modifier = Modifier.background(selectedColor),
-                    painter = it,
-                    contentDescription = null
-                )
+            Box(
+                modifier = Modifier
+                    .background(selectedColor)
+                    .wrapContentHeight(),
+                contentAlignment = Alignment.Center
+            ) {
+                painter?.let {
+                    Image(
+                        painter = it,
+                        contentDescription = null,
+                        contentScale = ContentScale.Inside
+                    )
+                }
+                messageTypeText?.let {
+                    Text(
+                        text = it.uppercase(),
+                        color = CustomTheme.textColors.darkText,
+                        fontWeight = FontWeight.Bold,
+                        style = CustomTheme.typography.regular,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.padding(top = 110.dp)
+                    )
+                }
```

Comment:
Let's not put this into the SelectableItem details. This already takes in "content" which is how we show the text below the image. 

At the time, the image was always the same type, but now that it can show different types of images we should pass in a "header" composable that does al that work for us. Then each different type of list item can pass in it's own type of header.

When we start passing in tons of params like this, things get very confusing and don't work properly. For instance, if I passed in a value for each param in this function, it would look broken. The function should be safe to use for all params.